core/services/chainlink/application.go

```diff
@@ -505,6 +508,20 @@ func (app *ChainlinkApplication) Start(ctx context.Context) error {
 
 	app.started = true
 
+	ticker := time.NewTicker(1 * time.Hour)
+	quit := make(chan struct{})
+	go func() {
+		for {
+			select {
+			case <-ticker.C:
+				pprof.Lookup("goroutine").WriteTo(os.Stdout, 1)
```

Comment:
Can you maybe print out a title line with a timestamp so it's easier to find stuff?